internal/transport/grpchttp2/framer.go

```diff
@@ -120,46 +117,29 @@ type Frame interface {
 // [DATA frame]: https://httpwg.org/specs/rfc7540.html#DATA
 type DataFrame struct {
 	hdr  *FrameHeader
-	free func()
-	Data []byte
+	Data *mem.Buffer
```

Comment:
We need a docstring here talking about whose responsible for calling Data.Free().

Here and in other frame types with a `mem.Buffer`.

Reply:
Done